test/verify/check-setroubleshoot

```diff
@@ -75,8 +75,20 @@ class TestSelinux(MachineCase):
 
         self.login_and_go("/selinux/setroubleshoot")
 
-        # at the beginning, there should be no entries
-        b.wait_in_text("body", "No SELinux alerts.")
+        if self.machine.image == "rhel-8-2-distropkg":
+            dismiss_sel = "button.pficon-delete"
+        else:
+            dismiss_sel = "#selinux-alert-dismiss"
+
+        # there are some distros with broken SELinux policies which always appear; so first, clean these up
+        while True:
+            # we either have no alerts, or an expand button in the table
+            b.wait_js_cond("ph_in_text('body', 'No SELinux alerts.') || ph_is_present('tbody .listing-ct-toggle')")
+            if "No SELinux alerts." in b.text('body'):
+                break
+            b.click('tbody .listing-ct-toggle')
```

Comment:
isn't this going to fail when there is more than one report with `element is ambiguous`?

Reply:
Right, we may need some `:first-child` there. Let me synthesize a second violation and try again.

Reply:
Indeed:

    RuntimeError: tbody .listing-ct-toggle:not([disabled]) is ambigous

Tested with
```diff
--- test/verify/check-setroubleshoot
+++ test/verify/check-setroubleshoot
@@ -73,26 +73,28 @@ class TestSelinux(MachineCase):
         self.allow_journal_messages(".*couldn't introspect /org/fedoraproject/Setroubleshootd: GDBus.Error:org.freedesktop.DBus.Error.NoReply: Remote peer disconnected.*")
         self.allow_authorize_journal_messages()
 
-        self.login_and_go("/selinux/setroubleshoot")
+        # fix audit events first
+        self.machine.execute(script=FIX_AUDITD)
 
         if self.machine.image == "rhel-8-2-distropkg":
             dismiss_sel = "button.pficon-delete"
         else:
             dismiss_sel = "#selinux-alert-dismiss"
 
+        self.machine.execute(script=SELINUX_ALERT_SCRIPT)
+        self.login_and_go("/selinux/setroubleshoot")
+        sit()
+
         # there are some distros with broken SELinux policies which always appear; so first, clean these up
         while True:
             # we either have no alerts, or an expand button in the table
             b.wait_js_cond("ph_in_text('body', 'No SELinux alerts.') || ph_is_present('tbody .listing-ct-toggle')")
             if "No SELinux alerts." in b.text('body'):
                 break
-            b.click('tbody .listing-ct-toggle')
+            b.click('tbody:first-of-type .listing-ct-toggle')
             b.click(dismiss_sel)
             b.wait_not_present(dismiss_sel)
 
-        # fix audit events first
-        self.machine.execute(script=FIX_AUDITD)
-
         #########################################################
         # trigger an alert
         self.machine.execute(script=SELINUX_ALERT_SCRIPT)

```